ui/src/assets/chat_page.scss

```diff
@@ -0,0 +1,110 @@
+/* Make the main container a flex column that fills the available height */
+.chat-container {
```

Comment:
You can move all this css into the plugin - just put it into a new file called `styles.scss` within your plugin's dir. It'll be included by default.